message/schemas/document-schema.avsc

```diff
@@ -233,99 +233,132 @@
+                          "name": "ProtocolMqtt311Type",
                           "fields": [
                             {
                               "type": "GenericRecord",
-                              "name": "message",
+                              "name": "protocoloptions",
                               "doc": "MQTT message metadata constraints"
                             }
                           ]
                         },
                         {
                           "type": "record",
-                          "name": "BindingMqtt50Type",
+                          "name": "ProtocolMqtt50Type",
                           "fields": [
                             {
                               "type": "GenericRecord",
-                              "name": "message",
+                              "name": "protocoloptions",
                               "doc": "MQTT message metadata constraints"
                             }
                           ]
                         },
                         {
                           "type": "record",
-                          "name": "BindingKAFKAType",
+                          "name": "ProtocolKAFKAType",
                           "fields": [
                             {
                               "type": "GenericRecord",
-                              "name": "message",
+                              "name": "protocoloptions",
                               "doc": "The Apache Kafka message metadata constraints"
                             }
                           ]
                         },
                         {
                           "type": "record",
-                          "name": "BindingHTTPType",
+                          "name": "ProtocolHTTPType",
                           "fields": [
                             {
                               "type": "GenericRecord",
-                              "name": "message",
+                              "name": "protocoloptions",
                               "doc": "The HTTP message metadata constraints"
                             }
                           ]
                         }
                       ],
-                      "doc": "Binding identifier. This attribute MUST be the same as the 'binding' attribute of the definition group, but is not automatically inherited because definitions may be cross-referenced across groups"
+                      "doc": "Protocol identifier. This attribute MUST be the same as the 'protocol' attribute of the definition group, but is not automatically inherited because definitions may be cross-referenced across groups"
                     },
                     {
                       "type": "string",
                       "name": "schemaformat",
                       "doc": "The schema format applicable to the message payload, equivalent to the 'format' attribute attribute of the schema registry"
```

Comment:
Was this deliberately not updated to "envelope format"? Also, I suspect "attribute attribute" is unintentional :)